go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/ceph/go-ceph
+
+go 1.12
+
+require (
+	github.com/gofrs/uuid v3.2.0+incompatible
```

Comment:
Where does the `+incompatible` come from?

Reply:
https://github.com/golang/go/wiki/Modules#can-a-module-consume-a-v2-package-that-has-not-opted-into-modules-what-does-incompatible-mean